tensorflow/python/keras/layers/wrappers_test.py

```diff
@@ -466,6 +466,13 @@ def test_TimeDistributed_with_ragged_input_with_batch_size(self):
         output_ragged, name='tensor')
     self.assertAllEqual(output_ragged.to_tensor(), output_dense)
 
+  def test_TimeDistributed_set_static_shape(self):
+    layer = keras.layers.TimeDistributed(keras.layers.Conv2D(16, (3, 3)))
+    inputs = keras.Input(batch_shape=(1, None, 32, 32, 1))
+    outputs = layer(inputs)
+    # Make sure the batch dim is not lost after array_ops.reshape.
+    self.assertEqual(outputs.shape, [1, None, 30, 30, 16])
```

Comment:
It seems that the test is failing since the tensor.shape in v2 is returning TensorShape type, rather than list. Can u update? Thanks

Reply:
This has been corrected.